silo-to-silo/contracts/src/SiloToSilo.sol

```diff
@@ -174,9 +172,13 @@ contract SiloToSilo is AccessControl {
 
     function _stringToUint(bytes memory b) private pure returns (uint128) {
```

Comment:
I propose adding this function to the SDK, I think this will useful

Reply:
I'm not sure this is the right place. This is parsing of a specific output result.....

Anyway, I think it should be in the separate PR.